File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Attestation.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Attestation{

	@SerializedName("playerAttestationRenderer")
	private PlayerAttestationRenderer playerAttestationRenderer;

	public void setPlayerAttestationRenderer(PlayerAttestationRenderer playerAttestationRenderer){
		this.playerAttestationRenderer = playerAttestationRenderer;
	}

	public PlayerAttestationRenderer getPlayerAttestationRenderer(){
		return playerAttestationRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Attestation.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class Attestation : Parcelable, Serializable {
    @SerializedName("playerAttestationRenderer")
    var playerAttestationRenderer: PlayerAttestationRenderer? = null
    override fun toString(): String {
        return "Attestation{" +
                "playerAttestationRenderer = '" + playerAttestationRenderer + '\'' +
                "}"
    }
}